ui/design-system-components/src/components/Entry/EntryCard/card-actions.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import Icon from '@akashaorg/design-system-core/lib/components/Icon';
+import Anchor from '@akashaorg/design-system-core/lib/components/Anchor';
+import Stack from '@akashaorg/design-system-core/lib/components/Stack';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+
+export type CardActionProps = {
+  profileId: string;
+  repliesAnchorLink?: string;
+  disableActions?: boolean;
+  isModerated?: boolean;
+  actionsRightExt?: React.ReactNode;
+  onReflect: () => void;
+};
+
+//@TODO: Refactor this component
+const CardActions: React.FC<CardActionProps> = props => {
+  const { profileId, repliesAnchorLink, disableActions, isModerated, actionsRightExt, onReflect } =
+    props;
+
+  if (isModerated) {
+    return (
+      <Stack
+        direction="row"
+        justify="end"
+        align="center"
+        spacing="gap-x-4"
+        padding="py-4"
+        customStyle="w-3/4 self-center"
+      >
+        <Button onClick={onReflect} plain>
+          <Icon type="ChatBubbleLeftRightIcon" accentColor={true} />
+        </Button>
+      </Stack>
+    );
+  }
+
+  return (
+    <Stack direction="row" align="center" justify="end" spacing="gap-x-4" padding="p-4">
+      <>{actionsRightExt}</>
+      <Anchor
+        href={`${repliesAnchorLink}/${profileId}`}
```

Comment:
is this href right? shouldn't be ${beamId} or something?

Reply:
Yes, it should be the `beamId`. Btw I have a todo for this component to refactor it that I didn't bother much about it.